.changeset/neat-tigers-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Allow configuring sitemap filenames generated by `@astro/sitemap`
```

Comment:
Changesets are meant to document and show new features. In your case, you want to explicitly discuss the new feature you're adding, why it was added, and how users can use it (code snippet). It's a minimal version of the docs.